packages/experimental/Shimmer/SPEC.md

```diff
@@ -0,0 +1,230 @@
+# Shimmer
+
+## Background
+
+The `Shimmer` component is a temporary animation placeholder for when a service call takes time to return data and we don't want to block rendering the rest of the UI.
+
+Many a time, applications must wait for content to load or perform lengthy data processing operations. During these times, it is vital to show the user that the application isn't stalled. Progress indicators like `Shimmer` are used in these cases.
+
+## Requirements
+
+If using FURN's theming, the `Shimmer` requires use of the `ThemeProvider` from `@fluentui-react-native/theme` to work properly with themes. Please see [this page](../../../docs/pages/Guides/UpdateThemeProvider.md) for information on updating your `ThemeProvider` if using the version from `@uifabricshared/theming-react-native`.
```

Comment:
We can consider adding the reference to current `ThemeReference` as well. Probably this - https://github.com/microsoft/fluentui-react-native/blob/main/docs/pages/Theming/Basics.md

Reply:
Modified